blocks/test/fixtures/core-embed.serialized.html

```diff
@@ -0,0 +1,6 @@
+<!-- wp:core/embed url="https://example.com/" -->
+<figure>https://example.com/
```

Comment:
I _think_ in some cases you can force it to accept whitespace with a technique using curly braces with a string:

```jsx
<span>Forced{ '\n' }Newline?</span>
```